axelrod/strategies/darwin.py

```diff
@@ -23,7 +23,14 @@ class Darwin(Player):
     """
 
     name = "Darwin"
-    memory_depth = float('inf')
+    behaviour = {
```

Comment:
Darwin "cheats" because it retains state between tournaments. I guess we don't have a dimension for that.